release.md

```diff
@@ -0,0 +1,53 @@
+# How to release using actions
+
+Releases are done from Github Actions.
+The overall process is to first tag the repo and then build it on Github Actions. The build will run tests, build binaries, generate a release in github and upload binaries. Continue to read about the crazyflie-release project for handling of full fw releases and release candidates.
+
+## 1. Preparing the candidate release
+1. Tag the crazyflie-firmware and the crazyflie2-nrf-firmware github repo with a release candidate tag in the form of “2022.03-rc1”
+1. In github: Collect issues of both repositories in the “next-release” milestone. 
+1. In github: Rename the milestones to the real release, for instance “2022.03”
+1. In github actions on both firmware's repos: 
+   * select the 'release' flow. Press the dow narrow on 'run workflow'
+   * select the release candidate tag (see screenshot under)
+   * press the green button 'Run Workflow'. 
+1. Wait until workflows has been finalized
+1. Make to two pre-releases of the two repositories public
+
+![](action_screenshot.png)
+
+## 2. Make a candidate release firmware zip folder
+1. Locally clone this repository (crazyflie-release)
```

Comment:
I would suggest to remove this part. As we move the build functionality to github, I think we should stop using the toolbelt release functionality (and remove it at some point). Just build the release candidate (in crazyflie-release) and use that for testing.